.build/config/cassandra_ci.yaml

```diff
@@ -0,0 +1,342 @@
+
+  - job: dtest-large
+    runtime_env: python
+    resources: {cpu=4, memory=16g, storage=32g}
+    description: Python based multi-node distributed tests with tests flagged as too large for the regular suite
+    env:
+      DTEST_ARGS: "--use-vnodes --num-tokens=${NUM_TOKENS} --only-resource-intensive-tests --force-resource-intensive-tests"
+  
+  - job: dtest-large-novnode
+    runtime_env: python
+    description: Python based multi-node distributed tests with tests flagged as too large for the regular suite with vnodes disabled
+    env:
+      DTEST_ARGS: "--only-resource-intensive-tests --force-resource-intensive-tests"
+  
+  - job: dtest-upgrade
+    parent: dtest
+    description: Python based multi-node distributed upgrade tests
+    multiplex: 25
+    env:
+      DTEST_ARGS: "--use-vnodes --num-tokens=${NUM_TOKENS} --execute-upgrade-tests --execute-upgrade-tests-only --upgrade-target-version-only --upgrade-version-selection all"
+  
+  - job: dtest-upgrade-large
+    parent: dtest
+    description: Python based multi-node distributed upgrade tests flagged as too large for the regular upgrade suite
+    env:
+      DTEST_ARGS: "--use-vnodes --num-tokens=${NUM_TOKENS} --execute-upgrade-tests --execute-upgrade-tests-only --upgrade-target-version-only --upgrade-version-selection all --only-resource-intensive-tests --force-resource-intensive-tests"
+  
+  #-----------------------------------------------------------------------------
+  # Longer running soak / burn / benchmarking tests
+  #-----------------------------------------------------------------------------
+  - job: stress-test
+    runtime_env: jvm
+    resources: {cpu=4, memory=1g, storage=5g}
+    description: Runs stress tests
+
+  - job: fqltool-test
+    runtime_env: jvm
+    resources: {cpu=4, memory=1g, storage=5g}
+    description: Tests to ensure cassandra-stress is operating correctly
+  
+  - job: microbench
+    runtime_env: jvm
+    resources: {cpu=4, memory=1g, storage=5g}
+    description: Microbenchmarks executing specific portions of the codebase
+  
+  - job: test-burn
+    runtime_env: jvm
+    resources: {cpu=4, memory=16g, storage=5g}
+    description: Long running and fuzz tests
+    test_timeout: 60000000
+
+  - job: long-test
+    runtime_env: jvm
+    resources: {cpu=4, memory=6g, storage=10g}
+    description: Execute long-running tests that are beyond the normal suite timeout
+    test_timeout: 600000
+    multiplex: 100
+
+  - job: cqlsh-test
+    runtime_env: jvm
+    resources: {cpu=4, memory=1g, storage=5g}
+    description: Subset of tests exercising cqlsh only
+    cmd: ./pylib/cassandra-cqlsh-tests.sh $(pwd)
+
+#-----------------------------------------------------------------------------
+# PARAMETERS
+#-----------------------------------------------------------------------------
+# Since JVM tests are generally quicker, we can afford to multiplex across more params
+jdk_parameters:
```

Comment:
these names are super not clear and confusion... this also feels like we could simplify the "processing" of this file and leverage yaml anchors?


```
shared_objects:
  - &jdk_version
     - 11
     - 17

...
job: unit
with_jdks: *jdk_version
```